host/schedule_test.go

```diff
@@ -457,7 +457,7 @@ func (s *scheduleIntegrationSuite) TestInput() {
 
 	_, err = s.engine.CreateSchedule(NewContext(), req)
 	s.NoError(err)
-	s.Eventually(func() bool { return atomic.LoadInt32(&runs) == 1 }, 4*time.Second, 200*time.Millisecond)
```

Comment:
4s to 5s seems marginal change, will that make big difference? should we just change to bigger value like 10s?

Reply:
The schedule is every 3s (same as the old cron tests), so 10s is more than one run. If we're concerned about flakiness on that scale, we can expand everything and use a longer interval. But then the elapsed run time will be longer, so we'll have to wait longer for all buildkite runs, etc.

It would be nice to not have to make this tradeoff, but we don't have a general time-skipping mechanism across components (or even within one). We could look into parallelizing integration tests to reduce elapsed time, which would ironically probably make things more flaky until we expand all the timeouts